source/common/http/http2/protocol_constraints.cc

```diff
@@ -1,23 +1,34 @@
 #include "common/http/http2/protocol_constraints.h"
 
 #include "common/common/assert.h"
+#include "common/http/utility.h"
+#include "common/protobuf/utility.h"
 
 namespace Envoy {
 namespace Http {
 namespace Http2 {
 
 ProtocolConstraints::ProtocolConstraints(
     CodecStats& stats, const envoy::config::core::v3::Http2ProtocolOptions& http2_options)
-    : stats_(stats), max_outbound_frames_(http2_options.max_outbound_frames().value()),
+    : stats_(stats), max_outbound_frames_(PROTOBUF_GET_WRAPPED_OR_DEFAULT(
```

Comment:
I'm confused. Why are these changes needed? Or was this a previous bug and unrelated to the flip?

/wait-any

Reply:
Good catch. These were left over work in progress. It just needed to fix CodecStreamIdleTimeout so that it provides fully initialized http2 options.